packages/store/src/stores/store-utils.js

```diff
@@ -1,7 +1,20 @@
 import { Far } from '@endo/marshal';
+import { M, matches } from '@endo/patterns';
 
 const { Fail, quote: q } = assert;
 
+/**
+ * @param {Passable} s
+ * @returns {s is CopySet}
+ */
+export const isCopySet = s => matches(s, M.set());
```

Comment:
does this differ from the one in Endo? https://github.com/endojs/endo/blob/4c8702a77923c4cf757718296d34d933b30b63a4/packages/patterns/src/keys/checkKey.js#L175

if so please clarify with a comment and maybe a rename

Reply:
Clarified with a comment and a TODO. PTAL